autoload/codefmt/fish_indent.vim

```diff
@@ -0,0 +1,59 @@
+" Copyright 2020 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+
+function! codefmt#fish_indent#GetFormatter() abort
+  let l:formatter = {
+      \ 'name': 'fish_indent',
+      \ 'setup_instructions': 'Install fish_indent (https://fishshell.com/docs/current/commands.html#fish_indent)' .
+          \ 'and configure the fish_indent_executable flag'}
```

Comment:
Missing a space at the boundary between lines: "…l#fish_indent)and configure".

Reply:
Done